script/Ciel/Day 3/03_01_CIEL03_2_3.txt

```diff
@@ -368,12 +389,16 @@ The news has been saying the same thing since the third case, but in the end, th
 [sha:017f88bcf40653dbea7bb8bea279da77b54d0348]{
 -- Page 86, Offset 5054.
 -- この現代日本でここまで殺人事件が続くなんて、異常を通り越して超常と言ってもいいレベルかもしれない。
-The fact these murders continue to occur to this extent in modern Japan is beyond abnormal and may even be considered paranormal.
+For a string of serial murders to continue this long in modern Japan is beyond abnormal.
+ It's practically bordering on paranormal.
 }
 [sha:2514f5dddb4e72f817a3a78a9d93e153d99ae18c]{
 -- Page 87, Offset 5055.
 -- 「……そうでした。現代の吸血鬼、なんてふざけたテロップがうたれてるから、身近な出来事の気がしなかったというか……たしかに軽率でした。俺だってこの街に住んでいるんだから、気をつけなくちゃいけないのに」
-"...You're right. The jokey banner on the news proclaiming it was the work of some 'modern vampire' made it seem like something I shouldn't concern myself with. But I live in this city too, so I should be taking it seriously."
+"...You're right.
+ I didn't really take it seriously after I saw the chyron calling it the work of a 'modern day vampire'...
```

Comment:
nobody knows what a chyron is, my browser thinks its a typo unless I make it a proper noun for the company name. revert it to "news banner" :aokoheh: